airbyte-integrations/connectors/source-mysql-v2/src/test/kotlin/io/airbyte/integrations/source/mysql/MysqlCdcIntegrationTest.kt

```diff
@@ -91,21 +88,6 @@ class MysqlCdcIntegrationTest {
                 stmt.execute("INSERT INTO test.tbl (k, v) VALUES (3, 'baz')")
             }
         }
-
-        val run2InputState: List<AirbyteStateMessage> = listOf(run1.states().last())
-        val run2: BufferingOutputConsumer =
-            CliRunner.source("read", config(), configuredCatalog, run2InputState).run()
-        // TODO: add assertions on run2 messages.
-
-        println()
-        println()
-        for (msg in run1.messages()) {
-            println(Jsons.valueToTree(msg))
-        }
-        println()
-        for (msg in run2.messages()) {
-            println(Jsons.valueToTree(msg))
-        }
```

Comment:
We should, in fact, have a `run2` in this test. The println's are dumb. I'm not against you removing these lines in this PR but I'd just like to know why.

Reply:
Oh, never mind this, I see the test structure has changed. This is fine. The tests all need to be revisited anyway once we have a connector that works.